Dockerfile

```diff
@@ -1,24 +1,53 @@
-FROM cgal/testsuite-docker:archlinux
+FROM cgal/testsuite-docker:ubuntu
 MAINTAINER Philipp Moeller <bootsarehax@gmail.com>
+RUN  apt-get update \
+  && apt-get install -y bison \
+     git \
+     flex-old \
+     graphviz \
+     python2.7 \
+     python-pyquery \
+     texlive-binaries \
+  && apt-get clean -y \
 
-RUN pacman -Syy && pacman -S --noconfirm \
-    bison \
-    flex \
-    git \
-    graphviz \
-    python2 \
-    python2-pyquery \
-    texlive-bin \
-    && \
-    pacman -Scc --noconfirm
+#RUN pacman -Syy && pacman -S --noconfirm \
+#    bison \
+#    flex \
+ #   git \
+ #   graphviz \
+ #   python2 \
+ #   python2-pyquery \
+ #   texlive-bin \
+ #   && \
+ #   pacman -Scc --noconfirm
```

Comment:
You know what I think about committing commented code: just remove the commented lines!